### ISSUE TITLE
no training, code finishes in few mins

### ISSUE THREAD

**Issue by bach-zouk** [YoungXIAO13/PoseFromShape#3]
Namespace(azi_classes=24, batch_size=16, dataset='ObjectNet3D', decrease=200, ele_classes=12, img_feature_dim=1024, inp_classes=24, keypoint=True, lr=0.0001, model=None, mutated=True, n_epoch=300, novel=False, num_render=12, pretrained_resnet=False, print_freq=50, random_range=0, shape='MultiView', shape_feature_dim=256, tour=2, workers=4)

Random Seed:  5015

after running, bash run/train_ObjectNet_3D.sh

**Comment by bach-zouk** [comment]
also, i see all scripts only use MV shape encoder. Is it possible to run with pointnet encoder?

**Comment by YoungXIAO13** [comment]
> Namespace(azi_classes=24, batch_size=16, dataset='ObjectNet3D', decrease=200, ele_classes=12, img_feature_dim=1024, inp_classes=24, keypoint=True, lr=0.0001, model=None, mutated=True, n_epoch=300, novel=False, num_render=12, pretrained_resnet=False, print_freq=50, random_range=0, shape='MultiView', shape_feature_dim=256, tour=2, workers=4)
> 
> Random Seed: 5015
> 
> after running, bash run/train_ObjectNet_3D.sh

Can you check the **err_log.txt** to see what is the error? And please make sure all the data are properly generated (eg. obj files and multiview rendering images) before training.

For the PointNet part, we'll release the code for training on ObjectNet3D.

**Comment by bach-zouk** [comment]
ok the issue was related to JPEG loading with library PILLOW. Downgrading it from 6.0 to 5.4.1 reolved the issue..Looking forward to the pointnet encoder code.
